javascripts/src/lib/format.spec.ts

```diff
@@ -19,4 +19,45 @@ describe('Format.dollarsToCents', () => {
     expect(Format.dollarsToCents("$1.2")).toBe(expectedAmount)
   )
 
+  test("properly throws on blank",() =>
+    expect(() => { Format.dollarsToCents("")}).toThrowError()
+  )
+
+  test("properly throws on null",() =>
+    expect(() => { Format.dollarsToCents(null)}).toThrowError()
+  )
+
+  test("properly throws on undefined",() =>
+    expect(() => { Format.dollarsToCents(undefined)}).toThrowError()
+  )
+
+})
+
+describe('Format.dollarsToCentsSafe', () => {
+
+  const expectedAmount = 120
```

Comment:
I don't remember why I picked that number in the original for dollarsToCents test. My hunch is that it was because it was mostly a random choice. That said, I suspect part of it was that it has a number with a trailing zero. Since dollar strings written as "1.20" and "1.2" would both be converted to 120, that would help test that corner case a bit more.

Reply:
Got it!